src/helper/service.cpp

```diff
@@ -66,14 +66,20 @@ service::ptr service::from_settings(const service_identifier &id,
 
     auto rc_client = remote_config::client::from_settings(
         id, rc_settings, std::move(products), std::move(capabilities));
+    bool const rc_available =
```

Comment:
OPINION WANTED! I didn't want to do the call to info within the threads and rather set the threads/polling once it's confirmed the agent supports RC. What's your opinion @Anilm3? Would you have chosen this place to do this check?

Reply:
Remote configuration isn't something static, the agent could be updated while appsec is running, the relevant thread should just continue calling poll and let the remote configuration client handle this internally.

1. On first poll, the remote configuration client should check if remote configuration is available through info:
    - If not available, it should check on every poll (or thereabouts).
    - If available it shouldn't check anymore.
2. If at some point remote configuration becomes unavailable, we revert back to 1.

Reply:
Can you check again plz